crates/swc_ecma_parser/src/parser/stmt.rs

```diff
@@ -1644,6 +1644,67 @@ let x = 4";
         );
     }
 
+    #[test]
+    #[should_panic(expected = "Expected 'from', got ','")]
+    fn issue_4369_1() {
+        test_parser(
+            r#"export * as foo, { bar } from "mod""#,
+            Syntax::Es(EsConfig {
+                export_default_from: false,
+                ..Default::default()
+            }),
+            |p| p.parse_module(),
+        );
+    }
+
+    #[test]
+    fn issue_4369_2() {
+        test_parser(
+            r#"export foo, * as bar, { baz } from "mod""#,
+            Syntax::Es(EsConfig {
+                export_default_from: true,
+                ..Default::default()
+            }),
+            |p| p.parse_module(),
+        );
+    }
+
+    #[test]
+    fn issue_4369_3() {
+        test_parser(
+            r#"export foo, * as bar from "mod""#,
+            Syntax::Es(EsConfig {
+                export_default_from: true,
+                ..Default::default()
+            }),
+            |p| p.parse_module(),
+        );
+    }
+
+    #[test]
+    fn issue_4369_4() {
+        test_parser(
+            r#"export * as bar, { baz } from "mod""#,
```

Comment:
I think this should panic (with tested error message)?

Reply:
Is there any spec of compound rules?
It's a little weird if we allow issue_4369_2, issue_4369_3, issue_4369_5 but disallow issue_4369_4.

I expects the default/namespace/name could be compounded arbitrarily if [export default from](https://github.com/tc39/proposal-export-default-from) proposal is accepted.

Reply:
I think this is not allowed at the moment, and I'm not sure if the proposal for `export default from` wil touch this one